### DISCUSSION TITLE
Database questions

### DISCUSSION THREAD

**Discussion by jimdbr** [Brewtarget/brewtarget#563] [Q&A] [answered]
I just merged the default database with my own database to pick up any additions. Before the merge I had one water profile, Burton on Trent. Now I have Burton on Trent and 14 empty profiles. At a quick glance my recipes are all OK.

How many items of each type are in the default database?

Thanks.

**Comment by matty0ung** [comment, marked as answer]
Sorry to hear the merge went a bit funny.  Glad it didn't corrupt anything.
To answer your specific question, you can download the default database from https://github.com/Brewtarget/brewtarget/blob/develop/data/default_db.sqlite and open it in, eg, DB Browser.  Eg shows 14 waters, 375 fermentables, 127 hops, 33 equipments (though some have "display" turned off).

**Reply by jimdbr** [reply]
I downloaded DB Browser version 3.11.2 and took a look at the default database that I downloaded. It looked OK so I merged again and everything is good. My working database also looks OK. I have no idea why the first merge attempt failed. There were no meaningful messages anywhere.

I build on Linux Mint 20.1.

Jim

**Reply by mikfire** [reply]
@jimdbr is there any chance you could provide me with your pre-updated database? I would like to believe this is an isolated problem, but I am really not that optimistic.

**Reply by mikfire** [reply]
I could recreate it, I've found the issue and am working on a fix.

The good news is no harm was done to your database. A restart of brewtarget would have resolved the issue.

**Reply by jimdbr** [reply]
Yes, my database and recipes look OK after a restart. I will try to recreate the problem here with one of the older database backups. My day job might delay that by a day or two.

**Comment by jimdbr** [comment]
I confirmed that fixing issue 564 resolved my original problem. I reloaded an older database that had one water profile (Burton on Trent) and merged default_db.sqlite. All water profiles loaded as expected.

Thanks for all your help.

**Comment by matty0ung** [comment]
Since https://github.com/Brewtarget/brewtarget/issues/564 is fixed, I'm archiving this discussion.
